project_generation/content/templates/api/api/api.go.tmpl

```diff
@@ -18,6 +18,7 @@ func Setup(ctx context.Context, r *mux.Router) *API {
 		Router: r,
 	}
 
+	log.Event(ctx, "remove hello endpoint")
```

Comment:
This should include a log level - `log.INFO`

Reply:
Thank you @daiLlew